pkg/addons/translate.go

```diff
@@ -25,6 +29,28 @@ const (
 	NestedFlagsBase64 NestedFlags = (1 << iota)
 )
 
+func translateAsset(o unstructured.Unstructured, cs *acsapi.OpenShiftManagedCluster, ext *extra) (unstructured.Unstructured, error) {
+	ts := Translations[KeyFunc(o.GroupVersionKind().GroupKind(), o.GetNamespace(), o.GetName())]
+	for _, tr := range ts {
+		var s string
+		if tr.F != nil {
+			s = tr.F(cs)
```

Comment:
in the future we can give tr.F the signature `func(*acsapi.OpenShiftManagedCluster) (string, error)` and if nil default it here to a closure wrapping util.Template, but this is fine for now.